092-self-healing-using-cruise-control.md

```diff
@@ -26,29 +29,29 @@ It would be useful for users of Strimzi to be able to have these anomalies fixed
 
 #### Anomaly Detector Manager
 
-The anomaly detector manager helps in detecting the anomalies and also handling the detected anomalies.
+The anomaly detector manager helps in detecting the anomalies as well as handling them.
 It acts as a coordinator between the detector classes as well as the classes which will be handling and resolving the anomalies.
-Various detector classes like `GoalViolationDetector`, `DiskFailureDetector`, `KafkaBrokerFailureDetector` etc. are used for the anomaly detection, which runs periodically to check if the cluster have their corresponding anomalies or not (This periodic check can be easily configured through the `anomaly.detection.interval.ms` configuration).
-Every detector class works in a different way to detect their corresponding anomalies. A `KafkaBrokerFailureDetector` will require a deep usage of requesting metadata to the Kafka cluster (i.e. broker failure) while if we talk about `DiskFailureDetector` it will be more inclined towards using the admin client API. In the same way the `MetricAnomalyDetector` will be making use of metrics and other detector like `GoalViolationDetector` and `TopicAnomalyDetector` will have their own way to detect anomalies.
+Various detector classes like `GoalViolationDetector`, `DiskFailureDetector`, `KafkaBrokerFailureDetector` etc. are used for the anomaly detection, which runs periodically to check if the cluster have their corresponding anomalies or not (The frequency of this check can be easily configured through the `anomaly.detection.interval.ms` configuration).
+Detector classes have different mechanisms to detect their corresponding anomalies. For example, `KafkaBrokerFailureDetector` utilises Kafka Metadata API whereas `DiskFailureDetector` and `TopicAnomalyDetector` utilises Kafka Admin API. Furthermore, `MetricAnomalyDetector` and `GoalViolationDetector` uses metrics to detect the anomalies.
 The detected anomalies can be of various types:
-* Goal Violation - This happens if certain optimization goals are violated (e.g. DiskUsageDistributionGoal etc.). These goals can be configured independently (through the `self.healing.goals` config) to those used for manual rebalancing..
+* Goal Violation - This happens if certain optimization goals are violated (e.g. DiskUsageDistributionGoal etc.). These goals can be configured independently (through the `self.healing.goals` configuration under the `spec.cruiseControl.config` section) to those used for manual rebalancing(goals configured in the KafkaRebalance custom resource).
```

Comment:
I think this part confuses me slightly, maybe can be reworded better. 
Is the below what you mean?
* Goal Violation - This happens if certain optimization goals such as `DiskUsageDistributionGoal` are violated.  The optimization goals can be configured independently from `KafkaRebalance` custom resource by using `spec.cruiseControl.config.self.healing.goals` in Kafka custom resource. 

When you say certain optimization goals, are you saying only some would trigger Goal Violation anomaly?

Reply:
I tried to make the sentence less confusing, I hope its better now